dev/com.ibm.ws.jaxrs.2.0_fat/fat/src/com/ibm/ws/jaxrs20/fat/RestMetricsTest.java

```diff
@@ -195,7 +213,7 @@ public void testCheckedExceptions() throws IOException {
 
         runGetCheckedExceptionMethod(CHECKED_METHOD_INDEX, 500, "/restmetrics/rest/restmetrics/checked/unmappedChecked", "Unmapped Checked");
 
-        ArrayList<String> metricsList = getMetricsStrings(METRICS_URL_STRING);
+        ArrayList<String> metricsList = getMetricsStrings(200,METRICS_URL_STRING);
```

Comment:
minor nitpick, but there should be a space after the comma

Reply:
done